apps/api/src/app/admin/admin.service.ts

```diff
@@ -332,12 +333,18 @@ export class AdminService {
     scraperConfiguration,
     sectors,
     symbol,
-    symbolMapping
+    symbolMapping,
+    url
   }: Prisma.SymbolProfileUpdateInput & UniqueAsset) {
+    const symbolProfileOverride = {
+      name: name as string,
+      assetClass: assetClass as $Enums.AssetClass,
```

Comment:
You can use the alias to simplify: `assetClass: assetClass as AssetClass,`

Reply:
Didn't know that! Changed